lib/client/browser-settings.js

```diff
@@ -2,6 +2,9 @@
 
 var _ = require('lodash');
 
+// VERSION 1 - 0.9.0 - 2015-Nov-07 - initial version
+var STORAGE_VERSION = 1;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['STORAGE_VERSION' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1285500130/issues/source?bid=2452379&fileBranchId=2945908#l6)